cnf-tests/testsuites/pkg/utils/consts.go

```diff
@@ -83,3 +83,25 @@ const (
 	// PtpOperatorConfigs contains the name of the ptp operator config CRD
 	PtpOperatorConfigs = "ptpoperatorconfigs.ptp.openshift.io"
 )
+
+const (
+	// N3000DaemonNamespace contains the name of the n3000 namespace
+	N3000DaemonNamespace     = "n3000-operators-resources"
+	// N3000DaemonNamespace contains the name of the n3000 deployment
+	N3000DeploymentName      = "n3000-controller-manager"
```

Comment:
If the namespace is configurable at deployment time, please make it overridable with an env variable

Reply:
Namespace is not configurable